rmc/pipeline/two_breaks/run_batches.py

```diff
@@ -172,10 +172,8 @@ def get_dpois_expr(
     :return: log10 of the probability under Poisson model.
```

Comment:
```suggestion
    :return: log of the probability under Poisson model.
```

Reply:
Similar change needed on line 164 (and same for corresponding 2 lines in `constraint.py`)